angular-workspace/projects/ni/nimble-angular/pipes/format-number-decimal.pipe.ts

```diff
@@ -0,0 +1,64 @@
+import { Inject, LOCALE_ID, Pipe, PipeTransform } from '@angular/core';
+import { DecimalUnitFormat } from '@ni/nimble-components/dist/esm/utilities/unit-format/decimal-unit-format';
+import { TableColumnNumberText } from '@ni/nimble-components/dist/esm/table-column/number-text';
```

Comment:
The alternate proposal will prevent this but FYI we should not depend directly on a component from a pipe. Any dependency on a component as a side-effect registers the component and in this case that means pulling in fast-foundation, etc all for a function that has no actual component dependency. That is a unique behavior of importing from a component and part of the reason we have separate `types` files to avoid accidentally depending on the component registration side-effect.